docs/tutorials/wiki2/definingmodels.rst

```diff
@@ -4,260 +4,195 @@
 Defining the Domain Model
 =========================
 
-The first change we'll make to our stock ``pcreate``-generated application will
-be to define a wiki page :term:`domain model`.
+The first change we'll make to our stock cookiecutter-generated application will be to define a wiki page :term:`domain model`.
 
 .. note::
 
-  There is nothing special about the filename ``user.py`` or ``page.py`` except
-  that they are Python modules.  A project may have many models throughout its
-  codebase in arbitrarily named modules.  Modules implementing models often
-  have ``model`` in their names or they may live in a Python subpackage of
-  your application package named ``models`` (as we've done in this tutorial),
-  but this is only a convention and not a requirement.
+    There is nothing special about the filename ``user.py`` or ``page.py`` except that they are Python modules.  A project may have many models throughout its codebase in arbitrarily named modules.  Modules implementing models often have ``model`` in their names or they may live in a Python subpackage of your application package named ``models`` (as we've done in this tutorial), but this is only a convention and not a requirement.
 
 
 Declaring dependencies in our ``setup.py`` file
-===============================================
+-----------------------------------------------
 
-The models code in our application will depend on a package which is not a
-dependency of the original "tutorial" application.  The original "tutorial"
-application was generated by the ``pcreate`` command; it doesn't know about our
-custom application requirements.
+The models code in our application will depend on a package which is not a dependency of the original "pyramidtut" application.  The original "pyramidtut" application was generated by the cookiecutter; it doesn't know about our custom application requirements.
 
-We need to add a dependency, the ``bcrypt`` package, to our ``tutorial``
-package's ``setup.py`` file by assigning this dependency to the ``requires``
-parameter in the ``setup()`` function.
+We need to add a dependency, the `bcrypt <https://pypi.python.org/pypi/bcrypt>`_ package, to our ``pyramidtut`` package's ``setup.py`` file by assigning this dependency to the ``requires`` parameter in the ``setup()`` function.
 
-Open ``tutorial/setup.py`` and edit it to look like the following:
+Open ``pyramidtut/setup.py`` and edit it to look like the following:
 
 .. literalinclude:: src/models/setup.py
-   :linenos:
-   :emphasize-lines: 12
-   :language: python
+    :linenos:
+    :emphasize-lines: 12
+    :language: python
 
 Only the highlighted line needs to be added.
 
+.. note::
+
+    We are using the ``bcrypt`` package from PyPI to hash our passwords securely. There are other one-way hash algorithms for passwords if bcrypt is an issue on your system. Just make sure that it's an algorithm approved for storing passwords versus a generic one-way hash.
```

Comment:
I rearranged this a bit. It flows better now, and only the first instance of bcrypt is hyperlinked.